Reflections/week 3 reflection.md

```diff
@@ -32,5 +32,4 @@ be prepared for the week after this one.
 
 # What resources will you use to accomplish your goals for the upcoming week? (please list out the resources)
 
-Youtube, stackoverflow, and documentation for respective libraries (I don't quite know which webcrawler I want to use 
-yet.)
\ No newline at end of file
+Youtube, stackoverflow, and documentation for respective libraries.
```

Comment:
Thanks for figuring out a way to still add the review option. I have read this weekly update in your files and I'll give feedback here.